packages/testcontainers/src/containers/RegTestContainer/index.ts

```diff
@@ -35,7 +35,8 @@ export class RegTestContainer extends DeFiDContainer {
       '-fortcanningheight=8',
       '-fortcanningmuseumheight=9',
       '-fortcanninghillheight=10',
-      '-fortcanningroadheight=11'
+      '-fortcanningroadheight=11',
+      '-greatworldheight=110'
```

Comment:
Once the v3 code is merged to the main branch, this code needs to be removed.
For now I set greatworldheight = `110`, updatemasternode does not work at block BEFORE 110 in this PR.